Source/WebCore/platform/graphics/avfoundation/objc/MediaPlayerPrivateMediaSourceAVFObjC.mm

```diff
@@ -1226,6 +1227,7 @@ void getSupportedTypes(HashSet<String>& types) const final
     auto player = m_player.get();
     if (player)
         player->firstVideoFrameAvailable();
+
```

Comment:
Nit: necessary?

Reply:
I like to organise in paragraph logical computational blocks..